src/queue/Queue.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * The queue package contains our Queue, QueueTest, QueueNode and QueueTest 
+ * classes.
+ * 
+ * @author Scott McDonald and Jon Cronin
+ */
+package queue;
+
+/**
+ * Queue implements the basic Queue ADT, with the functions enque() and deque().
+ * Users of this queue class can set the maximum capacity or accept the default
+ * of 10.
+ * @author Scott McDonald and Jon Cronin
+ */
+public class Queue {
+    private QueueNode head;
+    private QueueNode tail;
+    private int capacity;
+    private int size;
+    
+    // Define the return values the Queue class methods may use
+    public static final int Q_SUCCESS = 0;
+    public static final int Q_ERR_NULL = -1;
+    public static final int Q_ERR_FULL = -2;
+    
+    /**
+     * Default constructor. Sets head and tail to null, max capacity to 10 and
+     * current size to 0.
+     */
+    public Queue() {
```

Comment:
Yeah I was wondering if we should try and care. I'm fine either way. I've been trying to follow what's in "Core Java for the Impatient" which is on Safari. I think it's closer to the normal Java convention than the typical C convention, but I don't really mind either way.